### PR TITLE
Feature: select recording device for speech recognition

### PR DESCRIPTION
**Pull request by radical-equanimity**
Resolves mrbelowski/CrewChiefV4#196

### CODE REVIEW

File: CrewChiefV4.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
+# Visual Studio 15
```
Comment (TheIronWolfModding) [inline CrewChiefV4.sln]: please make sure solution is buildable and can be open on VS2013 or revert those lines.

File: CrewChiefV4/SpeechRecogniser.cs [hunk 2]
```diff
@@ -806,10 +831,10 @@ void sre_SpeechRecognized(object sender, SpeechRecognizedEventArgs e)
             }
             else if (voiceOptionEnum == MainWindow.VoiceOptionEnum.ALWAYS_ON)
             {
-                sre.RecognizeAsyncStop();
```
Comment (TheIronWolfModding) [inline CrewChiefV4/SpeechRecogniser.cs]: I think nAudio recording device should be conditional same as audio output.  Probably same preference will do

Comment (radical-equanimity) [reply CrewChiefV4/SpeechRecogniser.cs]: Why was NAudio made conditional in the first place? Is there some disadvantage to using it?

**Comment by radical-equanimity** [comment]
I agree that the naudio recognition should be optional and that GUID is a more dependable way to refer to the correct device. Appreciate the work you've put into this great project- thanks Jim!

**Comment by mrbelowski** [comment]
I've just had a half hour test session with this and it appears to be solid so once the rest of the crew have had a look at that new PR I'll merge it and credit you add you as a collaborator in the read me. Thanks for doing this dude, it's great :)

**Pull request closed without merging**